### ISSUE TITLE
[train] system prompt 사용하여 학습

### ISSUE THREAD

**Issue by ryangkyung-thaki** [ThakiCloud/LLaMA-Factory#5]
### 학습 하이퍼파라미터 최적화
- **배치 사이즈 증가**: 4 → 24 (6배)
- **Gradient accumulation**: 4 → 1
- **Effective batch size**: 64 → 96 (per_device × num_gpus × gradient_acc)

### 시스템 프롬프트 설정
- System prompt 활성화: `default_system: "You are a helpful assistant."`
